File: java/src/strategy/SimUDuckApp/Revision3/RubberDuck.java
package strategy.SimUDuckApp.Revision3;

public class RubberDuck extends Duck implements Quackable{

	public void display() {
		System.out.println("I'm a rubber duckie");

	}
	
	public void quack() {
		System.out.println("Squeak");
	}

}

File: src/strategy/SimUDuckApp/Revision2/RubberDuck.kt
package strategy.SimUDuckApp.Revision2

class RubberDuck : Duck() {
    override fun quack() {
        println("Squeak")
    }

    override fun display() {
        println("I'm a rubber duckie")
    }

    override fun fly() {
        println("I can't fly")
    }
}
